schema/collection.go

```diff
@@ -173,6 +173,7 @@ func buildSearchSchema(name string, queryableFields []*QueryableField) *tsApi.Co
 			Type:     s.SearchType,
 			Facet:    &s.Faceted,
 			Index:    &s.Indexed,
+			Sort:     &s.Sortable,
```

Comment:
Write the sort parameter downstream when creating collection schema in TS

Reply:
So as per their docs looks like for numeric fields it is enabled by default, so do we need to enable it in the schema?

> While sorting is enabled by default on numerical and boolean fields, sorting on a string field is only allowed if that field has sort property enabled in the collection schema

https://typesense.org/docs/0.23.1/api/search.html#sort-results

Reply:
Relying on their defaults can cause unexpected changes in future, if they change this behavior, it can go un-notice once we upgrade i.e. collections created post upgrade may have a different behavior. We should be explicit in our definition.

Reply:
I don't think we should worry about that because that will be a breaking change and it is okay to rely on these functionalities if anything changes then we can make it work for our forked version. But my main concern is when it is specified in the schema then that means they need to do additional work with each write(additional in-memory indexing structure) which means even though users may not be using this feature or don't even care about it there is an extra cost for maintaining it. 

Rather if we move the cost during the query time which means if a user has an intended use-case then they are aware that the query will be expensive because of sorting.